### DISCUSSION TITLE
Duplicate GitHub Pages Deployment

### DISCUSSION THREAD

**Discussion by paolobettelini** [community/community#22486] [Other Feature Feedback, Questions, & Ideas]
<p>I constantly push to a repository with GitHub Pages, each time I do so there’s a new deployment and the old one goes inactive. However, I noticed that I still have an active (old) deployment from Dec 3, 2021.<br>
I current have two GitHub Pages deployments, the current latest and the one from december.<br>
I don’t think it’s supposed to happen and I also think they the conflict with eachother, since some files that I pused after december are not found (return 404) on the site.<br>
I tried disabling and re-enabling Pages but the old deployment is still there.</p>
<p>How can I remove the old deployment?<br>
Thanks<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/b/eb4abe9e042e88003efc6c5ec9dea1236cdd5f3c.png" data-download-href="/uploads/short-url/xzuloDEtRzjVS1xtusFVxjdXRkU.png?dl=1" title="deployments" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/59686810/181071400-3ddf75d4-068b-41d0-a49a-a1f2e14848eb.png" alt="deployments" data-base62-sha1="xzuloDEtRzjVS1xtusFVxjdXRkU" width="472" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/b/eb4abe9e042e88003efc6c5ec9dea1236cdd5f3c_2_472x500.png, https://aws1.discourse-cdn.com/github/original/3X/e/b/eb4abe9e042e88003efc6c5ec9dea1236cdd5f3c.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/b/eb4abe9e042e88003efc6c5ec9dea1236cdd5f3c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/b/eb4abe9e042e88003efc6c5ec9dea1236cdd5f3c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">deployments</span><span class="informations">684×724 70 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Katsute** [comment, marked as answer]
<p>Try using this action: <a href="https://github.com/marketplace/actions/delete-deployment-environment" class="inline-onebox" rel="noopener nofollow ugc">Delete Deployment Environment · Actions · GitHub Marketplace · GitHub</a> it should clear all the active and inactive deployments</p>

**Comment by paolobettelini** [comment]
<p>Thank you, it worked!</p>
